doc/api/packages.md

```diff
@@ -323,6 +315,17 @@ package. It is not a strong encapsulation since a direct require of any
 absolute subpath of the package such as
 `require('/path/to/node_modules/pkg/subpath.js')` will still load `subpath.js`.
 
+All modern JS build tools support the `"exports"` field, but if necessary for
+compatibility with specific tooling or Node.js versions < 12.7.0, the `"main"`
+field can be included alongside `"exports"`:
```

Comment:
There are lots of projects stuck on non-modern build tools, like Webpack 4, which are hard to upgrade.

```suggestion
All supported versions of Node.js (since 12.7.0) and modern build tools
support the `"exports"` field. If a project is using an outdated version of
Node.js or a build tool, compatibility can be achieved by including the
`"main"` field alongside `"exports"`:
```

Reply:
Added with some adjustments.